spigot/src/main/java/com/sekwah/advancedportals/spigot/connector/container/SpigotServerContainer.java

```diff
@@ -61,4 +62,12 @@ public PlayerContainer[] getPlayers() {
                 .map(SpigotPlayerContainer::new)
                 .toArray(PlayerContainer[]::new);
     }
+
+    // Check if it's a material compatible with making portals
+    private boolean isAdvancedPortalBlock(Material material) {
+        return switch (material) {
+            case WATER, LAVA, AIR, NETHER_PORTAL, END_GATEWAY, END_PORTAL -> true;
```

Comment:
non full blocks also work, though these are the main ones. E.g. you can use iron bars or even stairs.

Though might be good to just include the main ones for now.